app/workers/sha_poller_worker.rb

```diff
@@ -0,0 +1,17 @@
+# Polls weather for subscribed locations
+class ShaPollerWorker
+  include Sidekiq::Worker
+  include Sidekiq::Repeat::Repeatable
+
+  repeat { hourly.minute_of_hour(0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55) }
+
+  def perform
+    logger.info "Running sha poll"
+
+    Helios2Schema.subscriptions.trigger(
+      "schemaShaPublished",
+      {},
+      AppEnv['HEROKU_SLUG_COMMIT']
+    )
```

Comment:
Does this create a connection with ActionCable? How often is this triggered/how does the trigger work?

Reply:
No, this is a sidekiq worker on the server side. The connection is continuous through a web socket between the server and its clients. The data value in the variable `HEROKU_SLUG_COMMIT` is broadcasted to the clients every 5 minutes.